front/lib/actions/mcp.ts

```diff
@@ -439,14 +446,40 @@ export class MCPConfigurationServerRunner extends BaseActionConfigurationServerR
     }
 
     const content = r.value;
+    const generatedFiles: ActionGeneratedFileType[] = [];
+    for (const c of content) {
+      let file: FileResource | null = null;
+      if (
+        c.type === "resource" &&
+        c.resource.mimeType &&
+        isSupportedFileContentType(c.resource.mimeType)
+      ) {
+        const r = await processAndStoreFromUrl(auth, {
```

Comment:
So these will be visible to other agents participating in the conversation you confirm?

Reply:
Yep I guess fine for now but to keep in mind.